app/models/user.rb

```diff
@@ -23,9 +23,18 @@ def to_param
   end
 
   def open_items
-    self.items.where(status: "open")
+    self.items.open.distinct
   end
 
+  def won_items
+    self.items.find_all { |item| self == item.high_bidder && item.closed? }.uniq
+  end
```

Comment:
we can talk about whether we want to stick with this .uniq / distinct method or whether we want to create an update action for bids if a user bids on the same item 
-probably depends most on what we think the admin wants to see